homeassistant/components/openweathermap/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/openweathermap",
   "iot_class": "cloud_polling",
   "loggers": ["pyopenweathermap"],
-  "requirements": ["pyopenweathermap==0.1.1"]
+  "requirements": ["pyopenweathermap==0.2.1"]
```

Comment:
Can we bump this in a separate preliminary PR?

Reply:
Will do, pending resolution of the [comment above](https://github.com/home-assistant/core/pull/128799#discussion_r1807821694) (None values) as will it require a new version.